[auth-docs] - Add Grafana Cloud Limitations (#69894)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/_index.md

## Grafana Auth

> Grafana OSS and Grafana Enterprise, of course has a built in user authentication system with password authentication enabled 

Reviewer: ```suggestion
Grafana OSS and Grafana Enterprise, has a built-in user authentication system with password authentication enabled 
```

> by default. For Grafana Cloud, the default authentication system uses your `grafana.com` account for authentication.  

Reviewer: ```suggestion
by default. For Grafana Cloud, the default authentication system uses your `grafana.com` account for authentication.  
```

If you change your organization name in the Grafana UI, this setting needs to be updated to match the new name.

> > Please Note: Anonymous authentication is not supported in Grafana Cloud.

Reviewer: ```suggestion
{{% admonition type="note" %}}
Anonymous authentication is not supported in Grafana Cloud.
{{% /admonition %}}

```

enabled = false
```

> > Please Note: Basic authentication (or Local Grafana Authentication) is not supported in Grafana Cloud.

Reviewer: ```suggestion
{{% admonition type="note" %}}
Basic authentication (or local Grafana authentication) is not supported in Grafana Cloud.
{{% /admonition %}}

```

disable_login_form = true
```

> > Please Note: For Grafana Cloud users, this option can only be configured under specific circumstances, e.g. when enabling LDAP Authentication, please contact Grafana Support for further assistance.

Reviewer: ```suggestion
{{% admonition type="note" %}}
For Grafana Cloud users, this option can only be configured under specific circumstances, such as enabling LDAP authentication. Please contact Grafana Support for further assistance.
{{% /admonition %}}

```
